### PR TITLE
Add Edit functionality to data bags and roles (Updated Screenshots)

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

This PR contains three major changes. The Data Bags and the Roles sections have been updated. With that, all the screenshots have been updated with respect to the newly updated Chef Automate UI.

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -85,6 +85,50 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
...
 
+The Chef Infra Server UI lets you:
+
+- Create a Role
+- List out all the roles at one place.
+- Search for a specific role from a list of roles.
+- View the details of roles:
+  - Run List
+  - Attributes
+- Delete a Role.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:96]: Use sentence punctuation consistently. Don't capitalize roles unless it is the first word in a sentence.

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 2]
```diff
@@ -85,6 +85,50 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
...
+
+#### Create a Role
+
+To create a new role, select **Create Role**, as shown below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:100]:
```suggestion
To create a new role, select **Create Role**:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 3]
```diff
@@ -85,6 +85,50 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
...
+
+{{< figure src="/images/automate/create-role-button.png" alt="Create Roles Button">}}
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:102]:
```suggestion
{{< figure src="/images/automate/create-role-button.png" alt="Select Create Roles Button">}}
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
A few suggestions and changes on roles. I'm assuming that this is the revised content from data bags.

**Review by kalroy — APPROVED** [review]

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 4]
```diff
@@ -85,6 +85,50 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
 
 [Roles]({{< relref "roles" >}}) let you define patterns and processes that exist across nodes in an organization as belonging to a single job function. Each role consists of zero (or more) attributes and a run-list.
 
+The Chef Infra Server UI lets you:
+
+- Create a Role
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md]:
```suggestion
- Create a role
```

File: components/docs-chef-io/content/automate/chef_infra_server.md (hunk 1 above)
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md]:
```suggestion
- Delete a role.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
These two shouldn't be capitalized.

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**